RELEASING.md

```diff
@@ -76,6 +76,9 @@ release is announced.
 1. Push the tag.
 1. Create a GitHub release on the Calico repo, upload the release file (calico_node/release-<VERSION>.tgz), and post the updated
    release notes (see above for details).
+1. Run `make update_canonical_urls OLD=vX.Y NEW=vX.Y` to switch the canonical URLs to the latest release version number. Pass in the number of the previous release via `OLD` and the number of the current latest release via `NEW`. Example: `make update_canonical_urls OLD=v3.0 NEW=v3.1`, where `3.0` was the previous latest and `3.1` is the new latest release.
+1. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed. 
```

Comment:
Can we be slightly more explicit by saying "Test the changes locally with `make htmlproofer`"?